app/views/two_factor_authentication/sms_opt_in/error.html.erb

```diff
@@ -0,0 +1,40 @@
+<%= image_tag asset_url('alert/fail-x.svg'),
+              alt: t('errors.alt.error'),
+              width: 54,
+              class: 'margin-bottom-2' %>
+
+<%= render PageHeadingComponent.new.with_content(t('two_factor_authentication.opt_in.title')) %>
+
+<p>
+  <%= t(
+        'two_factor_authentication.opt_in.opted_out_last_30d_html',
+        phone_number: content_tag(:strong, @phone_configuration.masked_phone),
+      ) %>
+</p>
+
+<p><%= t('two_factor_authentication.opt_in.wait_30d_opt_in') %></p>
+
+<% if @other_mfa_options_url %>
+  <h2 class="margin-top-6"><%= t('two_factor_authentication.opt_in.cant_use_phone') %></h2>
+
+  <%= link_to t('two_factor_authentication.login_options_link_text'),
+              @other_mfa_options_url %>
+<% end %>
+
+<%= render(
+      'shared/troubleshooting_options',
```

Comment:
More a UX question, but I sorta wonder if the "Can't use your phone?" fallback should be folded into the list of troubleshooting options.

Reply:
I think there's a case for keeping it where it is, because that one still keeps you in what you're doing, and the troubleshooting options are more like exit opportunities

Reply:
There's several places we use in-app alternative paths with a troubleshooting options list.

e.g.

IAL2 phone confirmation "use a different phone number"

![image](https://user-images.githubusercontent.com/1779930/152587143-ea440606-fa2b-4a71-9e4a-a76e288468db.png)

IAL2 "verify by mail instead"

![image](https://user-images.githubusercontent.com/1779930/152587092-32284584-7c63-4daf-97a0-dc12a2148a6e.png)

Reply:
Oh those are good points... 📡🦇🌕  @nickttng! If you have any thoughts on combining links for the error page

Reply:
@zachmargolis I think combining is a great way. :+1:

Reply:
Updated in 224597346d9e27bc53eeeab68838c05492a16bfd, error page looks like this now:
<img width="611" alt="Screen Shot 2022-02-04 at 1 14 33 PM" src="https://user-images.githubusercontent.com/458784/152604106-986ddc7d-5e31-48c5-8ed9-9a0652f00384.png">

Reply:
Looks good to me.

Reply:
@zachmargolis Should we update "Need immediate assistance? ..." to "Having trouble? ..."?

With the "Choose another authentication method" text now folded into the section, the "Need immediate assistance..." might not make sense with it.

Reply:
Sure, updated in 19bd04a65c4d6da889bb76bdafe8fed833010ab7
<img width="604" alt="Screen Shot 2022-02-04 at 1 39 38 PM" src="https://user-images.githubusercontent.com/458784/152606849-c9135dfa-11bf-4776-a76a-3dfa4e53381f.png">

Reply:
Looks good 👍🏼